File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
       Map<Integer, Integer> myMap = new HashMap<>();
       for(int i = 0; i < nums.length; i++){
        int num = nums[i];
        int complement = target - nums[i];
        if(myMap.containsKey(complement)){
            return new int[]{myMap.get(complement), i};
        }
        
            myMap.put(num, i);
       }
       return new int[]{};
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        i: int = 0
        j: int = len(numbers) - 1
        output = []
        
        while i < j:
            total = numbers[i] + numbers[j]
            if(total < target):
                i+=1
            elif(total > target):
                j-=1
            else:
                return [i + 1, j + 1]
        
   